From 0f46b817645f5a0b03a781957f05fe62bcf24e00 Mon Sep 17 00:00:00 2001
From: Juha Kallioinen <juha.kallioinen@jollamobile.com>
Date: Fri, 10 May 2013 15:57:18 +0000
Subject: [PATCH] Only accelerate python if explicitly requested

Python accel requires export SBOX_REDIRECT_FORCE=/usr/bin/python

Signed-off-by: Juha Kallioinen <juha.kallioinen@jollamobile.com>
---
 modes/obs-rpm-build+pp/fs_rules.lua | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/modes/obs-rpm-build+pp/fs_rules.lua b/modes/obs-rpm-build+pp/fs_rules.lua
index d5bae0c..306182e 100644
--- a/modes/obs-rpm-build+pp/fs_rules.lua
+++ b/modes/obs-rpm-build+pp/fs_rules.lua
@@ -86,10 +86,6 @@ python_bin_test = {
 	  map_to = target_root, readonly = true },
 	{ if_active_exec_policy_is = "Tools-python",
 	  map_to = tools, readonly = true },
-	{ if_exists_then_map_to = target_root, readonly = true },
-	{ if_exists_then_map_to = tools_root,
-	  log_level = "warning", log_message = "Mapped to tools_root",
-	  readonly = true },
 	{ map_to = target_root, readonly = true }
 }
 
@@ -114,7 +110,8 @@ accelerated_program_actions = {
 -- conditionally accelerated programs:
 -- check if file exists in target_root and only then try to accelerate it
 conditionally_accelerated_program_actions = {
-	{ if_exists_in = target_root, actions = accelerated_program_actions, protection = readonly_fs_always },
+	{ if_exists_in = target_root, actions = accelerated_program_actions,
+	  protection = readonly_fs_always },
 	{ map_to = target_root, protection = readonly_fs_always },
 }
 
-- 
1.8.3-rc3

